addons/website/static/tests/tours/snippet_popup_and_scrollbar.js

```diff
@@ -67,48 +68,62 @@ wTourUtils.registerWebsitePreviewTour("snippet_popup_and_scrollbar", {
     toggleBackdrop(), // show Popup backdrop
     checkScrollbar(false),
     {
-        content: "Close the Popup.",
-        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry",
+        content: "Close the Popup that has now backdrop.",
+        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry:first",
     },
     checkScrollbar(true),
     {
         content: "Open the Cookies Bar.",
-        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry:last-child",
+        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry:last",
     },
     checkScrollbar(true),
     toggleBackdrop(), // show Cookies Bar backdrop
     checkScrollbar(false),
     toggleBackdrop(), // hide Cookies Bar backdrop
     checkScrollbar(true),
     {
-        content: "Open the Popup.",
-        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry",
+        content: "Open the Popup that has backdrop.",
+        trigger: ".o_we_invisible_el_panel .o_we_invisible_entry:first",
     },
     checkScrollbar(false),
     wTourUtils.goBackToBlocks(),
+    {
+        content: "Check that 'Blocks' tab is active",
+        trigger: ".o_snippet_search_filter:not(.d-none)",
+        isCheck: true,
+    },
```

Comment:
This shouldn't be needed, as the trigger (snippet `s_media_list`) in next step is not supposed to exist / be found until this is done?